doc/development/guide_for_devs.rst

```diff
@@ -19,6 +19,7 @@ as well as the following Python packages for development purposes:
 * `networkx <https://networkx.org/>`_ >= 2.5
 * `NumPy <http://numpy.org/>`_ >= 1.21
 * `pandas <https://pandas.pydata.org/>`_ >= 1.2.1
+* `pylint <https://pypi.org/project/pylint/>`_ >= 2.13.9
```

Comment:
Yes, a good idea to add `pylint` here. Please add to `dev_requirements.txt` accordingly.